### PR TITLE
Add support for r2000ic_165f

### PR DESCRIPTION
**Pull request by jeritgeorge**
Added meshes and urdf files for the R2000 iC 165f robot

### CODE REVIEW

**Comment by marip8** [comment]
Looks good overall. There is some visible eccentricity in links 4 and 6 when they rotate, which means that the meshes were not exported with the origin close enough to their axes of rotation. Can you re-export those two links from Solidworks and make sure the origins are the correct distance from the ground plane (1.970 m per the datasheet)?

File: fanuc_r2000ic_support/urdf/r2000ic_165f.xacro [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" ?>
+<robot name="fanuc_r2000ic_165f" xmlns:xacro="http://wiki.ros.org/xacro">
```
Comment (gavanderhoorn) [inline fanuc_r2000ic_support/urdf/r2000ic_165f.xacro:2]: The naming convention is actually `fanuc_<series><variant>`, which in this case would be `fanuc_r2000ic165f` (no underscore between the series id and the variant).

The underscore is not needed, as Fanuc never (so far) has had a variant identifier which starts with a character (as you would then have a situation where it's unclear where the variant ID starts).

Comment (marip8) [reply fanuc_r2000ic_support/urdf/r2000ic_165f.xacro:2]: Thanks for the info; I'll revert that last commit

Two tangential questions:
1. Should I submit a PR with this model against the `fanuc_experimental` or `fanuc` repository if we plan on testing this model on hardware later this year or early next year?
1. Where can I get CAD models for the 125L variant? I made the URDF a while ago but haven't been able to find models online, and I don't have any contacts at Fanuc. The models I have for the 165F were sent to us by the integrator we're working with. I assume it probably wouldn't be appropriate to "scale" the 165F links to meet the 125L geometry

**Comment by gavanderhoorn** [comment]
> 1\. Should I submit a PR with this model against the `fanuc_experimental` or `fanuc` repository if we plan on testing this model on hardware later this year or early next year?

I actually also have a version of the R-2000iC/165F locally which I never committed.

I'll compare my version with the one you have here and if they match, we can merge either version.

We've had an `/165` for about 2 years now.

> 2\. Where can I get CAD models for the 125L variant? I made the URDF a while ago but haven't been able to find models online, and I don't have any contacts at Fanuc. The models I have for the 165F were sent to us by the integrator we're working with. I assume it probably wouldn't be appropriate to "scale" the 165F links to meet the 125L geometry

I've sent out a request. I'll let you know when I get something back.

In the meantime you *could* check your crc. I assume you guys have an account. Perhaps ask @JeremyZoss.

**Comment by gavanderhoorn** [comment]
@marip8: I've sent you an email.

**Pull request merged**